test/libsolidity/syntaxTests/viewPureChecker/inline_assembly_instructions_allowed_pure.sol

```diff
@@ -0,0 +1,90 @@
+contract C {
+    function f() public pure {
+        assembly {
+            stop()
+            pop(add(0, 1))
+            pop(sub(0, 1))
+            pop(mul(0, 1))
+            pop(div(0, 1))
+            pop(sdiv(0, 1))
+            pop(mod(0, 1))
+            pop(smod(0, 1))
+            pop(exp(0, 1))
+            pop(not(0))
+            pop(lt(0, 1))
+            pop(gt(0, 1))
+            pop(slt(0, 1))
+            pop(sgt(0, 1))
+            pop(eq(0, 1))
+            pop(iszero(0))
+            pop(and(0, 1))
+            pop(or(0, 1))
+            pop(xor(0, 1))
+            pop(byte(0, 1))
+            pop(shl(0, 1))
+            pop(shr(0, 1))
+            pop(sar(0, 1))
+            pop(addmod(0, 1, 2))
+            pop(mulmod(0, 1, 2))
+            pop(signextend(0, 1))
+            pop(keccak256(0, 1))
+            pop(0)
+            pop(mload(0))
+            mstore(0, 1)
+            mstore8(0, 1)
+            //pop(sload(0))
+            //sstore(0, 1)
+            //pop(gas())
```

Comment:
I did not check the actual current behaviour. But wouldn't it be better to not comment those out, so that we can see the error messages later?